backend_ta/src/main/resources/application-test.properties

```diff
@@ -5,4 +5,5 @@ spring.datasource.driver-class-name=org.h2.Driver
 spring.jpa.show-sql=false
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
 spring.jpa.properties.hibernate.format_sql=true
-spring.jpa.properties.hibernate.enable_lazy_load_no_trans=true
\ No newline at end of file
+spring.jpa.properties.hibernate.enable_lazy_load_no_trans=true
+request-debug=false
```

Comment:
I assume this is the  "disable authentication bypass" from the commit name, what does that mean exactly?